test/e2e-rh-sdl/rapidast/far/create_rapidast_configmap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
We might make the script more robust to errors setting `set -eou pipefail` just below the shebang, to catch non-zero exit status, unbound variables and errors in pipe commands.